include/xlnt/cell/cell.hpp

```diff
@@ -613,9 +608,9 @@ class XLNT_API cell
     bool operator==(const cell &comparand) const;
 
     /// <summary>
-    /// Returns true if this cell is uninitialized.
+    /// Returns false if this cell the same cell as comparand (compared by reference).
```

Comment:
Most reference types still provide value equality. I added the note here because I wasn't sure if the reference only comparison was intentional and value comparison would quietly change how existing code behaves

Reply:
Opened #317 to clarify